base/hosts.h

```diff
@@ -180,8 +180,7 @@ gvm_host_get_addr6 (const gvm_host_t *, struct in6_addr *);
 void
 gvm_host_add_reverse_lookup (gvm_host_t *);
 
-void
-gvm_host_free (gpointer);
+void gvm_host_free (gpointer);
```

Comment:
```suggestion
void
gvm_host_free (gpointer);
```

Reply:
I tried to revert this manually, but clang-format is asking for it. So, I leave it as it is.

Reply:
ok, too bad.